pom.xml

```diff
@@ -87,11 +96,34 @@
       <version>1.1.12.Final</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.arquillian.cube</groupId>
+      <artifactId>arquillian-cube-openshift</artifactId>
+      <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>io.undertow</groupId>
+          <artifactId>undertow-core</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <classpathDependencyExcludes>
+            <classpathDependencyExclude>org.arquillian.cube:arquillian-cube-openshift</classpathDependencyExclude>
```

Comment:
why is this required?

Reply:
Because Swarm uses Arquillian for unit tests as well. Without these excludes, Arquillian Cube would fire in the unit tests phase, which 1. is just wrong, 2. doesn't work because the image stream doesn't exist yet (`fabric8:build` wasn't executed yet), so Cube will wait indefinitely for the pod to be started.